style.css

```diff
@@ -0,0 +1,73 @@
+body{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    width: 240px;
+    margin: 100px auto;
+    height: auto;
+    border: 2px solid gainsboro;
+    background-color: gainsboro;
+    border-radius: 12px;
+    box-shadow: 5px 5px 15px -5px rgba(0, 0, 0, .3);
+}
+
+.display{
+    width: 90%;
+    margin: 30px auto 10px;
```

Comment:
Use relative positioning instead of absolute; an example is choosing `em` over `px`.

Reply:
I don't understand, kindly explain